composer.json

```diff
@@ -11,20 +11,21 @@
     }
     ],
     "require": {
-        "symfony/framework-bundle": "~2.7|~3.1|~4.0",
-        "symfony/finder": "~2.7|~3.1|~4.0",
-        "symfony/console": "~2.7|~3.1|~4.0",
-        "symfony/intl": "~2.7|~3.1|~4.0",
-        "symfony/translation":  "~2.7|~3.1|~4.0",
-        "symfony/twig-bundle": "~2.7|~3.1|~4.0"
+        "php": "^7.1",
+        "symfony/framework-bundle": "~3.4|~4.0|~5.0",
+        "symfony/finder": "~3.4|~4.0|~5.0",
+        "symfony/console": "~3.4|~4.0|~5.0",
+        "symfony/intl": "~3.4|~4.0|~5.0",
+        "symfony/translation":  "~3.4|~4.0|~5.0",
+        "symfony/twig-bundle": "~3.4|~4.0|~5.0"
     },
     "require-dev": {
-        "symfony/asset": "~2.7|~3.1|~4.0",
-        "symfony/yaml": "~2.7|~3.1|~4.0",
-        "symfony/browser-kit": "~2.7|~3.1|~4.0",
-        "symfony/twig-bundle": "~2.7|~3.1|~4.0",
-        "symfony/phpunit-bridge": "~2.7|~3.1|~4.0",
-        "phpunit/phpunit": "^4.8|~5.7"
+        "symfony/asset": "~3.4|~4.0|~5.0",
+        "symfony/yaml": "~3.4|~4.0|~5.0",
+        "symfony/browser-kit": "~3.4|~4.0|~5.0",
+        "symfony/twig-bundle": "~3.4|~4.0|~5.0",
+        "symfony/phpunit-bridge": "^3.4.24|^4.0|^5.0",
```

Comment:
The latest version of the PHPUnit bridge is compatible with PHP 5.5+ anyway. No need to support so many different versions IMO.